prow/cmd/phony/examples/test_issue.json

```diff
@@ -0,0 +1,258 @@
+{
```

Comment:
Both files work for issue. I was thinking that I needed another "test_issue.json" apart from "test_comment.json". I will remove the extra JSON file.